website/content/en/docs/deployment/rds-s3/guide.md

```diff
@@ -160,16 +188,129 @@ yq e '.rds.secretName = env(RDS_SECRET)' -i charts/common/aws-secrets-manager/rd
             {{< /tab >}}
             {{< /tabpane >}} 
 
+### 2.4 Manual S3 Setup
+   1. Export values:
+      ```bash
+      export S3_SECRET="<your s3 secret name>"
+      export S3_BUCKET="<your s3 bucket name>"
+      export MINIO_SERVICE_HOST=s3.amazonaws.com
+      ```
+
+   1. [IRSA ONLY] IRSA Roles
+
+      1. Associate an OpenID Connect (OIDC) provider with your IAM role to authenticate your cluster with the IAM service.
+
+         ```bash
+         eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
+         --region ${CLUSTER_REGION} --approve
+         ```  
+      2. Get the following OIDC information for future reference:
+         ```bash
+         export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+         export OIDC_PROVIDER_URL=$(aws eks describe-cluster --name $CLUSTER_NAME --region $CLUSTER_REGION \
+         --query "cluster.identity.oidc.issuer" --output text | cut -c9-)
+         ```
+
+      3. [Create an IAM policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html) to scope the permissions for the Role. For simplicity, we will restrict access to the specified S3_BUCKET as an example.
+         ```bash
+         printf '{
+         "Version": "2012-10-17",
+         "Statement": [
+               {
+                  "Effect": "Allow",
+                  "Action": "s3:*",
+                  "Resource": [
+                     "arn:aws:s3:::${S3_BUCKET}",
+                     "arn:aws:s3::::${S3_BUCKET}/*"
+                  ]
+               }
+            ]
+         }
+          ' > ./s3_policy.json
```

Comment:
you can also create the policy in this step to avoid one step of put-role-policy. See step 3 - https://surajkota.github.io/kubeflow-manifests/main/docs/add-ons/load-balancer/guide/#configure-load-balancer-controller

Reply:
when using aws-cli can't seem to provide both trust policy and another s3 policy in the same command will keep as now.